src/overlays/misc/ovl_kaleido_scope/z_kaleido_collect.c

```diff
@@ -68,7 +67,7 @@ void KaleidoScope_DrawQuestStatus(PlayState* play, GraphicsContext* gfxCtx) {
         0xFFCC, 0xFFCC, 0xFFCC, 0xFFCC, 0xFFCC,
     };
     static s16 sPlayedSongBtnsAlpha[SONG_MAX_LENGTH] = { 0 };
-    static s32 sUnused2 = 0;
+    static s32 sUnused = 0;
```

Comment:
isnt it clear that its the case here too?

Reply:
This variable not accessed OOB currently so I'm not sure. I'm curious, does it match as `static s16 sPlayedSongBtnsAlpha[SONG_MAX_LENGTH + 1]` without this unused variable, or does it require an extra s16 padding anyway?

Reply:
`sPlayedSongBtnsAlpha` is never indexed past 8

Even `sPlayedSongBtns` is never read past 8, it's only written to at index 8

Reply:
`sPlayedSongBtnsAlpha[SONG_MAX_LENGTH + 1]` with no unused variable does match, so it might be cleaner that way but idk which is more likely